scripts/test-transaction.sh

```diff
@@ -11,27 +11,27 @@ wallet1="wallet1.dat"
 client1="client1.dat"
 
 # Create wallet0 and save wallet id to var, mint
-wallet0_id=$($BUILD_DIR/src/uhs/client/client-cli $config_file $client0 $wallet0 mint 10 10 | grep -E '^[^\W]+$')
+wallet0_id=$("${BUILD_DIR}/src/uhs/client/client-cli" "${config_file}" "${client0}" "${wallet0}" mint 10 10 | grep -E '^[^\W]+$')
 
-echo "Wallet0_PK: $wallet0_id"
+echo "Wallet0_PK: ${wallet0}_id"
```

Comment:
wallet0_id is unused.
Should it be on line 16 "Wallet0_PK: ${wallet0}_id"?

I see wallet1_id on line 27.
On line 24 we have "Wallet1_PK: ${wallet1}_id", which makes me think line 16 is intentional ...

Reply:
ACK, thanks and fixed